packages/clerk-js/bundlewatch.config.json

```diff
@@ -2,7 +2,7 @@
   "files": [
     { "path": "./dist/clerk.js", "maxSize": "577kB" },
     { "path": "./dist/clerk.browser.js", "maxSize": "78kB" },
-    { "path": "./dist/clerk.headless.js", "maxSize": "51KB" },
+    { "path": "./dist/clerk.headless.js", "maxSize": "55KB" },
```

Comment:
I'm bumping this to +4 to not break too soon on the next PRs - let me know if you have any concerns!